tests/integration/test_charm.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+# Copyright 2023 Mariyan Dimitrov
+# See LICENSE file for licensing details.
+
+import asyncio
+import logging
+from pathlib import Path
+
+import pytest
+import yaml
+from pytest_operator.plugin import OpsTest
+
+logger = logging.getLogger(__name__)
+
+METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
```

Comment:
Should this have an encoding as per the new standard? https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#file-encoding

Reply:
It should, however these are tests generated by `charmcraft`, there are no tests written yet as part of the PoC.